tests/Tests/ORM/Functional/NewOperatorTest.php

```diff
@@ -1080,6 +1083,242 @@ public function testShouldSupportNestedNewOperators(): void
         self::assertSame($this->fixtures[1]->username, $result[1]['cmsUserUsername']);
         self::assertSame($this->fixtures[2]->username, $result[2]['cmsUserUsername']);
     }
+
+    public function testNamedArguments(): void
+    {
+        $dql = '
+            SELECT
+                new CmsUserDTONamedArgs(
+                    e.email,
+                    u.name,
+                    CONCAT(a.country, \' \', a.city, \' \', a.zip) AS address
```

Comment:
Please use [nowdoc](https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.nowdoc) so as to avoid this escaping.